ailab/benchmark/xdb.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+
+##############################################################################
+# Copyright 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+##############################################################################
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+import json
+
+# from .auth import context, interngraph_auth_params
+# from utils.custom_logger import getLogger
+# from utils.utils import requestsJson
```

Comment:
remove comments

Reply:
Removed the entire file from this commit.